index.html

```diff
@@ -50,6 +50,146 @@ <h3>Let's connect</h3>
+          <ul class="stack">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button class="btn">See Project</button>
+      </div> 
+      <div class="cards3">
+        <img src="./images/Snapshoot Portfolio-4.png" alt="p1" class="snm"/><br>
+        <h2 class="card-h2">Multi-Post
+          Stories</h2>
+        <ul class="lists">
+        <li class="head">CANOPY</li>
+          <IMG src="./images/Counter.png" alt="dot">
+          <li class="back">Back End Dev</li>
+        <img src="./images/Counter.png" alt="dot">
+          <li>2015</li>
+        </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="stack">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button class="btn">See Project</button>
+      </div> 
+      
+  
+  </section>
+  <section class="About-myself">
+    <div class="profile">
+    <h1 class="title">About<br>Myself</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <h3>Let's connect</h3>
+    <ul class="icons">
+       <li>
+        <a href="#"><img class="social-icons" src="./images/Vector.png" alt="twitter"></a>
+       </li>
+       <li>
+        <a href="#"><img class="social-icons" src="./images/Linkedin icon.png" alt="linkedin"></a>
+       </li>
+       <li>
+        <a href="#"><img class="social-icons" src="./images/m-vector.png" alt="more"></a>
+       </li>
+       <li>
+        <a href="#"><img class="social-icons" src="./images/g-vector.png" alt="github"></a>
+       </li>
+       <li>
+        <a href="#"><img class="social-icons" src="./images/p-vector.png" alt="peace"></a>
+       </li>
+    </ul>
+  <div id="resume">
+     <a href="#">
+        <button class="btn-resume">Get my Resume</button>
+     </a>      
+  </div>
+  <div>
+  <div class="about-right">
+    <div class="arrows" id="first-arrow">
+     <div id="languages">
+      <h2 id="heading-2">Languages</h2>
+      <i class="fa fa-angle-down"></i>
+      </div>
+    </div>
+      <ul class="language">
+        <li class="languages"> 
+          <img src="./images/Ellipse 1.png" alt="Javascript">
+          Javascript</li>
+        <li class="languagest">
+          <img src="./images/Ellipse 2.png" alt="HTML">
+          HTML
+        </li>
+        <li class="languages">
+          <img src="./images/Ellipse 3.png" alt="CSS">
+          CSS
+        </li>
+      </ul>
+    <div class="arrows">
+      <h2 id="heading-2">FrameWorks</h2>
+      
+    </div>
+    <hr>
+    <div class="arrows">
+      <h2 id="heading-2">Skills</h2>
+    </div>
+    <hr>
+    </div>
+  </div>
```

Comment:
- [ ] While you have really invested a lot of efforts into this, I believe you still a few more twists to turn before smashing it. Kindly observe the difference and make yours look like Figma's design 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/227690720-0ecbc25f-11d4-4ec6-8860-1719ac26325b.png)|![image](https://user-images.githubusercontent.com/99715640/227690826-87a92e3f-78ab-488d-9cff-f027bd1a8de3.png)|